package.json

```diff
@@ -151,6 +151,16 @@
                     "type": "boolean",
                     "description": "Use JavaScript source maps (if they exist)",
                     "default": true
+                },
+                "vscode-edge-devtools.autoAttachViaDebuggerForEdge": {
+                    "type": "boolean",
+                    "description": "Auto attach the Elements for Microsoft Edge extension when you launch a browser with the Debugger For Microsoft Edge debug adapter extension",
+                    "default": true
+                },
+                "vscode-edge-devtools.debugAttachTimeoutMs": {
+                    "type": "number",
+                    "description": "How many milliseconds to wait before attaching the Elements tool after launching a debugger",
+                    "default": 3000

```

Comment:
Why'd we go with 3000 here?

Reply:
I decided to change the attach model from a one time try after 3000ms, to a polling model similar to what the Edge debugger is doing. So the default is now 10seconds, but it is the amount of time to keep trying now, rather than the time to wait.